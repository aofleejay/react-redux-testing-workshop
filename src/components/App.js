import React, { Component } from 'react'
import CommentList from '../containers/CommentList'

const App = () => (
  <div className="container">
    <h1 id="app-title" className="title">Comment List</h1>
    <hr/>
    <CommentList />
  </div>
)

export default App
