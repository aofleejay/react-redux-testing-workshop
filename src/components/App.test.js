import React from 'react'
import { shallow } from 'enzyme'
import App from './App'

jest.mock('../containers/CommentList.js', () => 'CommentListContainer')

describe('Test App component', () => {
  it('Should match its snapshot', () => {
    const wrapper = shallow(<App />)

    expect(wrapper).toMatchSnapshot()
  })
})
