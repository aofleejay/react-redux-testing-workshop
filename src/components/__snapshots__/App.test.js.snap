// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test App component Should match its snapshot 1`] = `
<div
  className="container"
>
  <h1
    className="title"
    id="app-title"
  >
    Comment List
  </h1>
  <hr />
  <CommentListContainer />
</div>
`;
